### ISSUE TITLE
7.2.6 splash Android Chrome DOM Exception

### ISSUE THREAD

**Issue by blacktrash** [flowplayer/flowplayer#1375]
https://flowplayer.blacktrash.org/fp7/playlist-basic-js.html
```
Uncaught DOMException: Failed to read the 'buffered' property from 'SourceBuffer': This SourceBuffer has been removed from the parent media source.
    at e.onSBUpdateEnd
```
Could be https://github.com/video-dev/hls.js/issues/971, but demo uses hls.js 0.9.1 already, and it does not happen with flowplayer 7.2.5

**Comment by fvilpoix** [comment]
v0.10.0 of hls.js seems to fix this issue

https://github.com/video-dev/hls.js/releases/tag/v0.10.0

**Comment by blacktrash** [comment]
https://flowplayer.blacktrash.org/fp7/playlist-basic-js.html now uses hls.js v0.10.0

**Comment by webcast33** [comment]
Is there a work-around for this exception? It's an FP7 showstopper on Android devices. Thank you.

**Comment by nnarhinen** [comment]
Most likely the workaround from #1376 works here too

**Comment by blacktrash** [comment]
No, [as mentioned](https://github.com/flowplayer/flowplayer/issues/1376) it only helps with 7.2.5

**Comment by nnarhinen** [comment]
I fixed a similar error yesterday with the `hls.destroy()` trick on `load` event.

**Comment by blacktrash** [comment]
https://flowplayer.blacktrash.org/fp7/playlist-basic-js.html uses the trick with 7.2.6 now, and fails.
No problem with the same setup and 7.2.5.

**Comment by nnarhinen** [comment]
Same with this - is this reproducable with latest dev?

**Comment by blacktrash** [comment]
Yes. https://flowplayer.blacktrash.org/fp7/playlist-basic-js.html now uses dev build

**Comment by nnarhinen** [comment]
https://github.com/video-dev/hls.js/issues/1787 most likely this

**Comment by blacktrash** [comment]
Latest dev:
<img width="1000" alt="screen shot 2018-07-13 at 11 52 04 am" src="https://user-images.githubusercontent.com/168338/42688273-8694c5a2-8693-11e8-9b48-552c5865cdd3.png">

I'm gonna stick with 7.2.5 and this hack

```js
flowplayer.engine('hlsjs-lite').plugin(function (o) {
  o.player.on('load', function () {
    o.hls.destroy();
  });
});
```
then (same hls.js library as in demo).

**Comment by nnarhinen** [comment]
https://flowplayer.blacktrash.org/7.2.6/flowplayer-dev.js is not at latest dev

**Comment by blacktrash** [comment]
Sorry, forgot `git reset` after fetching. - Yes, seems to work now, great!

**Comment by webcast33** [comment]
I've been tracking the progress on this issue because it's causing a significant problem for me. Using 7.2.6, I cannot play any videos on an Android device with Chrome because of the SourceBuffer exception. It's not a playlist issue.  Because I prefer to stick with the splash method, I'm using 7.2.5 instead. But there is a bug in 7.2.5 that prevents error events from propagating (so if a video file doesn't exist, the "spinner" just spins with no error message).

I think a resolution for this problem is scheduled for 7.2.7.  I'm sure a lot of Flowplayer users are similarly affected. Do you have a time frame for a fix? Thank you.
